frontend/app/src/components/Sidebar/Sidebar.test.tsx

```diff
@@ -348,7 +348,7 @@ describe("Sidebar Component", () => {
         "stLogo"
       )
       expect(sidebarLogo).toHaveStyle(
-        `max-width: calc(${sidebarWidth} - 5.75rem)`
+        `max-width: calc(${sidebarWidth} - 2 * 1.5rem - 0.5rem - 2.25rem)`
```

Comment:
nit: could you please leave a comment where this formular is coming from?

Reply:
There is a comment in styled-components where this is defined. But I added the comment here as well.